test/integration/commands/bundle_test.exs

```diff
@@ -61,4 +61,8 @@ defmodule Integration.Commands.BundleTest do
     assert_error_message_contains(response, "Unknown subcommand 'not-a-subcommand'")
   end
 
+  test "installing a bundle", %{user: user} do
+    [payload] = send_message(user, "@bot: operable:bundle install heroku 0.0.4")
+    assert %{name: "heroku", version: "0.0.4"} = payload
```

Comment:
I think we should try to use the views when we can... that way we have one central place to do the JSON conversion, and the Cog commands look just like the API requests. Documentation for one can double as documentation for the other.

And if /when we ever get around to calling embedded commands via the API, it'll be purely a backend change, without any user-visible effects.